Inpsyde/PhpcsHelpers.php

```diff
@@ -510,7 +525,18 @@ public static function isNullReturn(File $file, int $returnPosition): bool
         $nextToReturn = $file->findNext([T_WHITESPACE], $returnPosition, null, true, null, true);
         $nextToReturnType = $tokens[$nextToReturn]['code'] ?? '';
 
-        return $nextToReturnType === T_NULL;
+        if ($nextToReturnType !== T_NULL) {
+            return false;
+        }
+
+        $returnPosition = $nextToReturn + 1;
+        $followedBySemicolon = ($tokens[$returnPosition]['code'] ?? '') === T_SEMICOLON;
+
+        if (!$followedBySemicolon) {
+            return false;
+        }
+
+        return true;
```

Comment:
`followedBySemicolon`: I thought reading the variable names the reading get facilitated since we want to known to if there ware a valid return statement.

Ok I'll change it.

Regarding the space I was thinking you refered to the space after the return but not the one before the end statement. 

Is there a reason why I want to have a space between the `null` and the `;`?
When we search using `findNext` excluding `[T_WHITESPACE, T_NULL]` means the `T_NULL` must be present otherwise the returned value will be incorrect?

Reply:
When you says

> we make sure that it works also for things like return null; which makes the PR code fail...

Trying a fixture for 

``` 
function hh(): void {
    return null;
}
```

I missing something?